libgearman-server/struct/gearmand.h

```diff
@@ -75,7 +74,7 @@ struct gearmand_st
 
     void keepalive_idle(int keepalive_idle_)
     {
-      _keepalive= true;
+      keepalive(1);
```

Comment:
Is `_keepalive= true` bad coding style according to PVS? It should be more efficient than `keepalive(1)` unless the compiler can optimize it.

Reply:
the commit with the changes is removed